tracer/test/snapshots/AspNetCoreIisMvc21MvcResourceBasedSamplingTests._.verified.txt

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /ping,
+    Service: Samples.AspNetCoreMvc21,
+    Type: web,
+    Tags: {
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 200,
+      http.url: http://localhost:00000/ping,
+      http.useragent: testhelper,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      version: 1.0.0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.rule_psr: 0.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: -1.0
+    }
```

Comment:
FYI, the fix changes the snapshots from the following:

```
     Metrics: {
       process_id: 0,
       _dd.top_level: 1.0,
       _dd.tracer_kr: 1.0,
       _sampling_priority_v1: 1.0
     }
```


to what you see now. i.e.

```diff
     Metrics: {
       process_id: 0,
+      _dd.rule_psr: 0.0,
       _dd.top_level: 1.0,
       _dd.tracer_kr: 1.0,
-      _sampling_priority_v1: 1.0
+      _sampling_priority_v1: -1.0
     }
```